api/managementpb/backup/locations.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+// FSConfig represents file system configuration.
+message FSConfig {
+  string path = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+// S3Config represents S3 bucket configuration.
+message S3Config {
+  string endpoint = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  string access_key = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  string secret_key = 3 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+// Location represents single Backup Location.
+message Location {
+  // Machine-readable ID.
+  string location_id = 1;
+  // Location name
+  string name = 2;
+  // Short description
+  string description = 3;
+  oneof config {
+    FSConfig fs_config = 4;
+    S3Config s3_config = 5;
+  }
+}
+
+message ListLocationsRequest {}
+
+message ListLocationsResponse {
+  repeated Location locations = 1;
+}
+
+message AddLocationRequest {
+  // Location name
+  string name = 2 [
```

Comment:
why not starting from `1`?

Reply:
Had same qustion :D
https://github.com/percona/pmm/blob/release/2.15/api/managementpb/ia/channels.proto#L109
This was my inspiration, so not sure if it is intended.

Reply:
Some time ago there was the`channel_id ` field :D

https://github.com/percona/pmm/blob/a91001eeaee379944aa4df4c7a7aa1d2db1f7623/api/managementpb/ia/channels.proto#L106

I think you can start with 1.